docs2/site/docs/migrations/migration5.md

```diff
@@ -84,6 +84,27 @@ and `GraphQL.NewtonsoftJson` libraries, these serialize and deserialize to JSON
 
 ### 6. `AutoRegisteringObjectGraphType` and `AutoRegisteringInputObjectGraphType` enhancements
 
+These two classes now provide a range of customizable behavior for data models without the
+need for creating individual graph types for each data model. New for v5, fields' names
+and graph types can be customized by applying attributes to the respective property, such
```

Comment:
only property (not method) ?

Reply:
for auto registering input types, yes only property.  Future PR will add "method" in here as it would apply to output classes.

Reply:
ok